### PR TITLE
Create templates for issues and pull requests

### PR DESCRIPTION
**Pull request by C0DK**
I saw that you lacked templates for the different github features (pullrequests and issues) so i wanted to quickly help out as i love the service. I might have made a few typos, but i hope they work. Please supply any corrections and i'll gladly change the PR :smiley:  

It solves issue #668 

## Checklist
*sadly all irrelevant*
- [x] The code change is tested and works locally.
- [x] Tests pass. Your PR cannot be merged unless tests pass
- [x] There is no commented out code in this PR.
- [x] Have you followed the guidelines in our Contributing document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you written new tests for your core changes, as applicable?

### CODE REVIEW

**Comment by wren** [comment]
Thank you! I've been meaning to put this in place. I have time set apart this weekend to work on jrnl, so I'll review this then.

**Comment by C0DK** [comment]
> Thank you! I've been meaning to put this in place. I have time set apart this weekend to work on jrnl, so I'll review this then.

I really love jrnl - so i wanted to contribute somehow, and that felt like an easy place to start :)

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  - [ ] support request
+
+
+
+
+## Regarding bug reports:
+
+* **Please tell us about your environment:** 
+  - Jrnl version: (run `jrnl -v`)
+  - Python version: (run `python -V`)
```
Comment (maebert) [inline .github/ISSUE_TEMPLATE.md]: Note that `jrnl` might be using a different python than the one that `python` points to. The quickest way to find it is to run ``head -1  `which jrnl` ``, but we should probably just print this when running `jrnl -v` instead.

Comment (C0DK) [reply .github/ISSUE_TEMPLATE.md]: I'd argue that'd be a thing `jrnl -v` should print. i.e: `jrnl version 1.9.8 (running on python3.8.1)` kindof.

Comment (C0DK) [reply .github/ISSUE_TEMPLATE.md]: I've added this to the `jrnl -v` output now :)

**Comment by C0DK** [comment]
So I've changed the output of version, so it's more fulfilling - But the test cases checking that fails. so I don't know if you want to accept the new format and with it change the test case

**Comment by wren** [comment]
@C0DK , can you please separate the version thing from the rest of the PR. We're trying to have one fix per PR to facilitate code review, and scheduling feature and bug releases.

**Comment by C0DK** [comment]
@wren will do :) sorry.

**Comment by wren** [comment]
@C0DK Thanks, again! I'd be happy to take a look at the version thing in a separate PR, if you want to resubmit it.

**Pull request merged**